docs/org-units/getting-web-api.ipynb

```diff
@@ -0,0 +1,257 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4c8e54f2",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: Getting org units from DHIS2 Web API\n",
+    "short_title: Getting from Web API\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef0392bd",
+   "metadata": {},
+   "source": [
+    "This page shows practical ways to fetch organisation unit (OU) data from the DHIS2 Web API using `dhis2-python-client` library.\n",
+    "\n",
+    "In the **climate-tools** environment, `dhis2-python-client` library is already installed. You only need to come up with your DHIS2 credentials and server address to use this library. Below is a basic usage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "519e4d4f",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "▶ Current DHIS2 version:  2.42.2-SNAPSHOT\n"
+     ]
+    }
+   ],
+   "source": [
+    "from itertools import islice\n",
+    "import pprint\n",
+    "\n",
+    "from dhis2_client import DHIS2Client\n",
+    "from dhis2_client.settings import ClientSettings\n",
+    "\n",
+    "# Client configuration\n",
+    "cfg = ClientSettings(\n",
+    "  base_url=\"http://localhost:8080\",\n",
+    "  username=\"admin\",\n",
+    "  password=\"district\")\n",
+    "\n",
+    "client = DHIS2Client(settings=cfg)\n",
+    "info = client.get_system_info()\n",
+    "\n",
+    "# Check if everything is working.\n",
+    "# You should see your current DHIS2 version info.\n",
+    "print(\"▶ Current DHIS2 version: \", info[\"version\"])"
```

Comment:
Multiple args to print(arg1, arg2, etc) are automatically separated by space. Remove the space after "version:" to avoid the double space.